whois-commons/src/main/java/net/ripe/db/whois/common/rpsl/DummifierNrtmV4.java

```diff
@@ -21,13 +21,20 @@ public class DummifierNrtmV4 extends DummifierNrtm {
         DUMMIFICATION_REPLACEMENTS.put(AttributeType.REMARKS, "Dummified");
     }
 
+    private final Set<ObjectType> writtenPlaceHolders = Sets.newHashSet();
```

Comment:
Similar to code in whois-scheduler `DecorationStrategy` class but it's OK for me to create separate logic here instead of trying to re-use the original code.